corehq/apps/app_manager/views/utils.py

```diff
@@ -294,7 +294,7 @@ def update_linked_app_and_notify(domain, app_id, user_id, email):
     subject = _("Update Status for linked app %s") % app.name
     try:
         update_linked_app(app, user_id)
-    except AppLinkError as e:
+    except (AppLinkError, MultimediaMissingError) as e:
```

Comment:
Should `MultimediaMissingError` be re-raised, like in the `except Exception` block, so that the process crashes?

Reply:
@orangejenny we know what the error is so we just send the email to the user whereas in Exception we want to be notified what went wrong.
Does that sound right?

Reply:
Yeah, that makes sense. I had been thinking that we should block the pull from succeeding, but thinking about it again that isn't necessary, since we can re-pull the multimedia.